jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -304,6 +304,15 @@ def generate_volume_path(self, mountPath, default_mount_path, volume_name):
         return mountPath
       return os.path.join(default_mount_path, mountPath)
     return os.path.join(default_mount_path, volume_name)
+
+  @classmethod
+  def get_mem_limit(self, memory_limit):
```

Comment:
How does this handle values like Ki and Ti? I tested with Ki and it seems that it passed into the env var as a string.

![image](https://user-images.githubusercontent.com/60748071/110950843-a38eed80-8344-11eb-9873-2c59a12cf25f.png)

Reply:
That said, do we need to handle values like that?

Reply:
We can add those, not a big deal, my thought was that the only 3 options we'd see normally is Gi, Mi, nothing..but I am happy to add those